Cython/Compiler/ExprNodes.py

```diff
@@ -6976,11 +6976,7 @@ def analyse_as_ordinary_attribute_node(self, env, target):
         self.obj = self.obj.analyse_types(env)
         self.analyse_attribute(env)
         if self.entry and self.entry.is_cmethod and not self.is_called:
```

Comment:
Maybe we should only set `entry.is_cgetter` and not `entry.is_cmethod` for these properties ? If `@propertiy` is used, I think it's intended to make the entry non-callable, i.e. not a method.

Reply:
it is still callable, just as a c-function not a method. I tried setting `is_cmethod =0` when `is_cgetter = True` and it broke the tests, requiring more logic everywhere `is_cmethod` is checked.